test/cypress/integration/fixtures/user-data.json

```diff
@@ -2,16 +2,16 @@
   "admin": {
```

Comment:
I think we can delete and ignore this file (gets regenerated on every test run)

Reply:
That's a good point 🙇‍♂️